source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -176,6 +184,12 @@ You can select and initialise a specific component by using its `data-module` at
   </script>
 ```
 
+When initialised individually, errors are thrown rather than logged.
+This will stop the execution of the script and you'll need to either:
+
+* fix what causes the error
+* use a [`try/catch` block](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch) to change the behaviour of your code in reaction to the error
```

Comment:
With the section above saying "You must check your application works without errors" we don't need to repeat ourselves and talk about fixing errors here too

```suggestion
When initialised individually, errors are thrown rather than logged and must be handled with a [`try/catch` block](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch).
```

Reply:
Thinking we benefit from a little duplication of the reminder to fix errors here for people getting sent a link to that section specifically (and who would not have seen the previous mention).

Reply:
Yeah that sounds good

The current wording says they can pick either fixing errors or adding `try…catch` but not both

How about we duplicate the one-liner above but change _should_ to _must_?

```suggestion
You must check your application works without errors or some components will not work correctly.
```

<br>

We can always mention the `try…catch` as an optional safety feature rather than something that’s mandatory. For example, their test coverage might not hit every scenario where an error is thrown.

Reply:
Brill, thanks for keeping it simple